draft-ietf-webpush-encryption.md

```diff
@@ -279,32 +279,25 @@ application server MUST set the `rs` parameter in the `aes128gcm` content coding
 header to a size that is greater than the length of the plaintext, plus any
 padding (which is at least 2 octets).
 
-A push message MUST include a zero length `keyid` parameter in the content
-coding header.
+A push message MUST include the application server ECDH public key in the
+`keyid` parameter of the encrypted content coding header.
```

Comment:
Since this is the MUST mandate, would it make sense to move the "The uncompressed point...of the "keyid"." language on lines 187-189 of this diff here?